### ISSUE TITLE
Allow Tab through the buttons from ShareButtons component for Keyboard Accessibility

### ISSUE THREAD

**Issue by dongmei-cao** [MurhafSousli/ngx-sharebuttons#278]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?

All OS/Versions

### Versions

Angular CLI: 1.6.5
Node: 8.9.4
OS: darwin x64
Angular: 
...

### Repro steps

Latest code from the repo

### Desired functionality

Able to Tab through the media buttons from the `ShareButtons` component. In order to do this, each <share-button> should contain `tabindex="0"` attribute.

**Comment by silentsnooc** [comment]
@dongmei-cao Why don't you just set it?

**Comment by MurhafSousli** [comment]
So you think we should have `tabindex="0"` on all buttons or `[tabindex]="i"` ?

**Comment by dongmei-cao** [comment]
@MurhafSousli Should be tabindex="0". You do not want to explicitly defined the tabbing order. The tabindex="0" just means the element will receive keyboard focus.

**Comment by dongmei-cao** [comment]
@MurhafSousli I digged into more on the component. Actually, you don't need to add the `tabindex="0"`. The `<button>` element should receive focus by default. But it is not visually showing the focus because the .`sb-wrapper `has outline: 0. I'm not sure why the `outline` is overridden. But it needs to be there for `:focus.`

**Comment by MurhafSousli** [comment]
@dongmei-cao  The outline prevents the selection of buttons text.

**Comment by MurhafSousli** [comment]
@dongmei-cao The button actually receives the focus, you just need to add a focus style using CSS
```css
.sb-wrapper {
  &:focus {
    border: 1px solid blue;
  }
}
```
